src/ratings/entities/rating.entity.ts

```diff
@@ -0,0 +1,34 @@
+import {
+  Column,
+  CreateDateColumn,
+  Entity,
+  ManyToOne,
+  PrimaryGeneratedColumn,
+  UpdateDateColumn,
+} from 'typeorm';
+import { Wine } from '../../wines/entities/wine.entity';
+import { IsInt, Max, Min } from 'class-validator';
+
+@Entity()
+export class Rating {
```

Comment:
I am pretty sure we should save the author of the review. for authentication and for displaying purposes. And to also get all the ratings a user submitted

Reply:
done